tests/fixtures/typescript-types

```diff
@@ -1,6 +1,8 @@
 {number}
 {{ object: with, comma: separator }}
 {{ object: with, trailing: comma, }}
+{{ object: with; semicolon: separator }}
+{{ object: with; semicolon: separators; }}
```

Comment:
probably should test mixed separators since the parser supports them

Reply:
I've added a test with mixed commas/semi-colons, and can uncomment the test I added for newlines in the newline PR